packages/eslint-plugin/ROADMAP.md

```diff
@@ -1,10 +1,10 @@
 ﻿# Roadmap
 
-✅ (28) = done  
```

Comment:
I don't know the consensus but maybe is time to replace the two spaces with `<br />` ?

Reply:
@ficristo would you rather I do that in this PR or a second one?

I'm happy for either, but would prefer a second PR to keep this and #194 clean.